src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/HttpContextHelpers.cs

```diff
@@ -0,0 +1,37 @@
+#if NETFRAMEWORK
+using System;
+using System.Web;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.Headers;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.Integrations.AspNet
+{
+    internal static class HttpContextHelpers
```

Comment:
The usual naming is singular: `HttpContextHelper`

Reply:
Sounds good, renamed to `HttpContextHelper` in 59123778f187fd5034b83c472b32b53278f5b1e4